01x_lazy.ipynb

```diff
@@ -412,6 +412,17 @@
     "Similarly, we can launch Python processes and threads in the background. Some methods allow mapping over multiple inputs and gathering the results, more on that later.  The thread starts and the cell completes immediately, but the data associated with the download only appears in the queue object some time later."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Edit sources.py to configure source locations\n",
```

Comment:
@martindurant Is this what you mean by letting the user know where the URL came from?

Reply:
Was this already there? Yes, and in that case sorry! :)

Reply:
:) no worries. Thanks!